File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_vip_user_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusVipUserVO:
    """VIP会员用户信息VO"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    user_id: int = None
    vip_level_id: int = None
    status: str = None
    point_balance: int = None
    total_recharged_points: int = None
    valid_from: str = None
    valid_to: str = None
    last_active_time: str = None
    remark: str = None
    vip_level: PlusVipLevelVO = None

File: sdkwork-backend-sdk-go/types/plus_vip_user_vo.go
package types

// VIP会员用户信息VO
type PlusVipUserVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	UserId int `json:"userId"`
	VipLevelId int `json:"vipLevelId"`
	Status string `json:"status"`
	PointBalance int `json:"pointBalance"`
	TotalRechargedPoints int `json:"totalRechargedPoints"`
	ValidFrom string `json:"validFrom"`
	ValidTo string `json:"validTo"`
	LastActiveTime string `json:"lastActiveTime"`
	Remark string `json:"remark"`
	VipLevel PlusVipLevelVO `json:"vipLevel"`
}
